sdk/python/plexi_sdk/_app.py

```
@@ -13,6 +13,48 @@
+            limit: Arg[int] = Arg("--limit", type=int, default=100)
+            count: Arg[int] = Arg(positional=True, type=int, default=10)
+
+            async def on_init(self, ctx):
+                print(self.repo_dir)  # already resolved
+    """
+
+    def __init__(
+        self,
+        *flags: str,
+        positional: bool = False,
+        type: Any = None,
```

Ruff (0.15.14)

[error] Function argument `type` is shadowing a Python builtin

(A002)